actions/errata.py

```diff
@@ -98,7 +98,12 @@ def old_update(errataidlist, cache_only=None):
 
     return packages.update(packagelist, cache_only)
 
-def update(errataidlist, cache_only=None):
+def update(params, cache_only=None):
+    allow_vendor_change = False
+    if 'allow_vendor_change' in params and params['allow_vendor_change']:
+        allow_vendor_change = True
+
+    errataidlist = params['errata_id']
```

Comment:
This label was just changed

```suggestion
    errataidlist = params['errata_ids']
```